### ISSUE TITLE
Cannot work properly with multiple monitors.

### ISSUE THREAD

**Issue by richex-cn** [NickeManarin/ScreenToGif#393]

**Comment by richex-cn** [comment]
Thank you very much for your reply.

There are two monitors on my desk, which cannot select the secondary monitor area on my left when intercepting the area. It is recommended to test this situation on your computer.

The reason I found this problem was that I wanted the ScreenToGif to record on the left side of the monitor, so that it would not affect my right side of the monitor while doing other work. But I found I couldn't set up the recording area on the left monitor.

I'm not very good at English. If I don't understand something, I'm willing to cooperate with you to know more details.

My display Settings are like this:
![20181027_093623](https://user-images.githubusercontent.com/25103518/47598264-31fcb180-d9cc-11e8-9543-064b5f5041f5.png)

System version:
Windows 10 1803
